clang/unittests/Format/FormatTestMacroExpansion.cpp

```diff
@@ -255,6 +255,44 @@ TEST_F(FormatTestMacroExpansion,
                          Style);
 }
 
+TEST_F(FormatTestMacroExpansion, CommaAsOperator) {
+  FormatStyle Style = getGoogleStyle();
+  Style.Macros.push_back("ASSIGN_OR_RETURN(a, b, c)=a=(b); if(x) c");
+  verifyFormat(R"(ASSIGN_OR_RETURN(auto reader,
```

Comment:
It's hard to read the test cases with raw strings. Can you set `ColumnLimit` to 60 so that we don't have to use raw strings?